Makefile

```diff
@@ -11,34 +11,18 @@ else
 OPENSSL_DIR ?= /usr
 endif
 
-include .requirements
 .PHONY: install dev lint test test-integration test-plugins test-all fix-windows
 
+KONG_NETTLE_VERSION ?= `grep KONG_NETTLE_VERSION .requirements | awk -F"=" '{print $$2}'`
```

Comment:
Since we removed nettle from `.requirements` (because unused in this repository), can we also remove mentions of it from the Makefile as well?